### PR TITLE
Env updates PR

### PR DESCRIPTION
**Pull request by PaLeroy**

### CODE REVIEW

**Comment by moratodpg** [comment]
I have added now a tutorial for generating the transition model. WIP

File: imp_env/pomdp_models/pomdp_new.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    self.T0[i,j,j]=1
+                else:
+                    H, _ = np.histogram(Dnext, self.d_interv) 
+                    self.T0[i,j,:] = H/countd.sum()
+        self.T0[-1,] = self.T0[-2,]
+
+        self.Tr = np.zeros((det_rates, n_dstates, n_dstates))
+        self.Tr = np.tile(self.b0,(det_rates, n_dstates,1))                
+            
+        return self.d_interv
```
Comment (PaLeroy) [inline imp_env/pomdp_models/pomdp_new.py]: d_interv never updated?

File: imp_env/pomdp_models/generate_pomdp.ipynb [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+       "       [7.24397570e-02, 9.27560243e-01]])"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model_dr.O"
```
Comment (PaLeroy) [inline imp_env/pomdp_models/generate_pomdp.ipynb]: AttributeError: 'Pomdp' object has no attribute 'O'

**Comment by PaLeroy** [comment]
I had some errors, I reported the first one.
A good practice is to restart your notebook for the last tests and also to clean the output before committing the file since it reduces the number of changes to look at in GitHub.

**Comment by moratodpg** [comment]
Thanks for the comments.
I renamed some variables and solved the issues with the tutorial!

**Pull request merged**